test/integration/examples/apiserver_test.go

```diff
@@ -489,6 +494,161 @@ func TestAggregatedAPIServerRejectRedirectResponse(t *testing.T) {
 	}
 }
 
+func TestAggregatedAPIServerIdentity(t *testing.T) {
```

Comment:
This test might be better in `test/integration/controlplane/apiserver_identity_test.go`

Reply:
I would prefer to avoid exporting the test logic related to getting an aggregated API server started.  It is rather messy.